dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestCriteria.cpp

```diff
@@ -897,7 +1184,7 @@ TEST(DataInterfaceIIDMTest, testLoadCriteriaLocalValue) {
     criteria6.addLoad(loads[i]);
   ASSERT_FALSE(criteria6.empty());
   ASSERT_FALSE(criteria6.checkCriteria(0, false));
-  ASSERT_EQ(criteria6.getFailingCriteria().size(), 1);
+  ASSERT_GT(criteria6.getFailingCriteria().size(), 0);
```

Comment:
Why does this size change? the failing criteria vector is used by dynawo-algorithms project, it should not be empty

Reply:
The current test checks that criteria6.getFailingCriteria() isn't empty.